ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogCollection.kt

```diff
@@ -42,4 +43,9 @@ class MogCollection(
   fun getMogBoundsForTile(tileCoordinates: TileCoordinates): TileCoordinates =
     if (tileCoordinates.zoom < hiResMogMinZoom) TileCoordinates.WORLD
     else tileCoordinates.originAtZoom(hiResMogMinZoom)
+
+  private fun String.mustContainParam(param: String): String {
```

Comment:
Nit: `checkContains` would be closer to Kotlin check naming convention. This function doesn't have any logic specific to the idea of URL params, so I would remove it from the name.

You could also model this as `checkContainsAll(vararg substrings: String)` and remove the method chaining. The function could then also define as a single expression function.